src/github.com/getlantern/lantern-android/client/client.go

```diff
@@ -110,18 +104,11 @@ func (client *MobileClient) ServeHTTP() {
 // updateConfig attempts to pull a configuration file from the network using
 // the client proxy itself.
 func (client *MobileClient) updateConfig() error {
-	var err error
 	var buf []byte
-	var cli *http.Client
-
-	if cli, err = util.HTTPClient(cloudConfigCA, client.Addr); err != nil {
-		return err
-	}
```

Comment:
Oh so it's not quite as bad as I thought -- I thought it was not proxying to the config at all, but it's just not doing DDF.